src/components/DetailComponent/DetailComponent.js

```diff
@@ -0,0 +1,253 @@
+import { Container, Grid } from "@material-ui/core";
+import React, { Fragment, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import { IMAGE_PATH } from "../../store/constants";
+import Typography from "@mui/material/Typography";
+import Loader from "../../shared/loader";
+import { detailStyle } from "../../styles/appStyles";
+
+export default function DetailComponent({
+  detailData,
+  detailReq,
+  detailLoader,
+  reviews,
+  error,
+  match,
+}) {
+  const classes = detailStyle();
+  const routesData = match.params;
+  useEffect(() => {
+    window.scrollTo(0, 0);
+    detailReq(routesData);
+    console.log(reviews);
```

Comment:
Remove that.

Reply:
Removed and added missing dependencies in useEffect